village/src/components/SmurfForm.js

```diff
@@ -18,6 +19,16 @@ class SmurfForm extends Component {
   addSmurf(event) {
     event.preventDefault();
     // add code to create the smurf using the api
+    const { name, age, height } = this.state;
+    console.log(name, age, height);
+    axios 
+    .post('http://localhost:3333/smurfs', { name, age, height })
+    .then(response => {
+      this.setState({ smurfs: response.data });
```

Comment:
Again, not sure why we're keeping Smurf data in `smurfs` array in this component